PostHog/PostHogFeatureFlags.swift

```diff
@@ -55,10 +55,8 @@ class PostHogFeatureFlags {
         var recordingActive = true
 
         // check for boolean flags
-        if let linkedFlag = sessionRecording["linkedFlag"] as? String,
-           let value = featureFlags[linkedFlag] as? Bool
-        {
-            recordingActive = value
+        if let linkedFlag = sessionRecording["linkedFlag"] as? String {
+            recordingActive = featureFlags[linkedFlag] as? Bool ?? false
```

Comment:
@ioannisj we should do something similar for the next check too:
L64 most likely

Reply:
`featureFlags[linkedFlag]` can also be a string, and that means its enabled, this is the case for multivariant, linked with "any"

Reply:
See PR [PostHog/posthog-android@`2929c4d` (#230)](https://github.com/PostHog/posthog-android/pull/230/commits/2929c4de1b808ef026d1b1e77dd401d6617564ca)